src/Helpers/functions.php

```diff
@@ -6,3 +6,8 @@ function consoleOutput(): ConsoleOutput
 {
     return app(ConsoleOutput::class);
 }
+
+function isSleepHelperAvailable()
```

Comment:
Let's just update the minimum required version for the package, so we can dispatch this function and all it's usages.

Reply:
Alright, I've just removed it